integration/angular_cli/browserslist

```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
Previously this contained only `last 2 Chrome versions` so that during "tests large" we don't run differential loading for each browser test.

Reply:
`packages/angular_devkit/build_angular/test/hello-world-app/.browserslistrc` still contains `last 2 Chrome versions`, and that's the one that browser tests use.